### ISSUE TITLE
Bot front-end

### ISSUE THREAD

**Issue by Solobrosco** [Solobrosco/monitor-bot#15]
## Open for discussion
To add to our full-stack we need a front-end.
Question is, what should be a suitable interface for is bot?

**Comment by aaronvanoung** [comment]
I was thinking of something simple and user friendly. It could have a simple drop down to choose which store we want to search then a input section to search an item. we can also but a section where it prints out checking and then when found can say in stock. then we would stop the loop and return the handle for the site.

**Comment by Solobrosco** [comment]
Not 2020 no more but this website helps https://steelkiwi.com/blog/top-10-python-web-frameworks-to-learn/

**Comment by Solobrosco** [comment]
django and flask are pretty popular

**Comment by Solobrosco** [comment]
I also want to add - something that is niche and not everyone is using

**Comment by Solobrosco** [comment]
This is gonna make us stand out if we down to put in the work
https://isg-one.com/research/isg-provider-lens/articles/edge-computing-the-new-paradigm-in-iot

**Comment by aaronvanoung** [comment]
Wait so by using flask is this bot accessible online ? I was thinking this bot would be something that just runs on your personal computer.

**Comment by aaronvanoung** [comment]
Actually i think I like that idea to use django and flask, would definitely expose me to more useful tools and we can have a place to actually present this project and show its functionality

**Comment by Solobrosco** [comment]
Yeah if that's the direction we should go, users can monitor stuff online. Then again having a program on ur computer that monitors sites would be cleaner than connecting ppl to this service. But yeah I feel like having an interface would help a lot instead of looking at the terminal screen. I think python has some sort of GUI we can use, haven't tried it that's why I kinda suggested a frontend library
